### PR TITLE
Fix dshot bidir switch

### PR DESCRIPTION
**Pull request by Asizon**
Alternative to https://github.com/betaflight/betaflight-configurator/pull/2816

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — DISMISSED** [review]

File: src/js/tabs/motors.js
Comment (haslinghuis) [inline src/js/tabs/motors.js:691]:
```suggestion
                        buttonNoCallback: null,
```

File: src/js/tabs/pid_tuning.js
Comment (haslinghuis) [inline src/js/tabs/pid_tuning.js:465]:
```suggestion
                        buttonNoCallback: null,
```

File: src/js/tabs/motors.js
Comment (haslinghuis) [inline src/js/tabs/motors.js:694]: Can we put this function in `fc.js` as it used in two tabs now? Including `dialog[DynFilter]Settings`
But is up to you to decide.

Comment (Asizon) [reply src/js/tabs/motors.js:694]: I think fc.js only have fc related settings, not a good place.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by ctzsnooze** [comment]
I tested this with 4.2.11 and I do not get the dialog, nor do the dynamic notch settings change, when I
- turn DShot telemetry on, or off
- turn RPM filtering on or off, while DShot Telemetry is off.

**Comment by ctzsnooze** [comment]
Thanks, Asizon, for explaining why we are not 'fixing' the behaviour for 4.2.  So I withdraw my previous concern.

**Comment by Asizon** [comment]
Fixed @ctzsnooze thanks for testing!

**Review by ctzsnooze — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by ctzsnooze** [comment]
Tested - works really well!  Thanks @Asizon

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — DISMISSED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by Asizon** [comment]
Removed outdated and confusing comments.

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by ctzsnooze — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
